features/admin.connections.v1/resources/guides/google/quick-start.tsx

```diff
@@ -267,103 +268,19 @@ const GoogleQuickStart: FunctionComponent<GoogleQuickStartPropsInterface> = (
-                                            }
-                                        ] }
-                                        filterAttributePlaceholder={
-                                            t("applications:advancedSearch.form." +
-                                                "inputs.filterAttribute.placeholder")
-                                        }
-                                        filterConditionsPlaceholder={
-                                            t("applications:advancedSearch.form." +
-                                                "inputs.filterCondition.placeholder")
-                                        }
-                                        filterValuePlaceholder={
-                                            t("applications:advancedSearch.form." +
-                                                "inputs.filterValue.placeholder")
-                                        }
-                                        placeholder={ t("applications:" +
-                                            "advancedSearch.placeholder") }
-                                        defaultSearchAttribute="name"
-                                        defaultSearchOperator="co"
-                                        triggerClearQuery={ triggerClearQuery }
-                                        data-testid={ `${ testId }-list-advanced-search` }
-                                    />)
-                                }
-                                currentListSize={ appList.count }
-                                listItemLimit={ listItemLimit }
-                                onItemsPerPageDropdownChange={ handleItemsPerPageDropdownChange }
-                                onPageChange={ handlePaginationChange }
-                                showPagination={ appList?.totalResults > listItemLimit }
-                                totalPages={ Math.ceil(appList.totalResults / listItemLimit) }
-                                totalListSize={ appList.totalResults }
-                                data-testid={ `${ testId }-list-layout` }
-                                showTopActionPanel={ appList?.totalResults > listItemLimit }
-                                paginationOptions={ {
-                                    itemsPerPageDropdownLowerLimit: ITEMS_PER_PAGE
-                                } }
-                            >
-                                <ApplicationList
-                                    isSetStrongerAuth
-                                    list={ appList }
-                                    onEmptyListPlaceholderActionClick={
-                                        () => history.push(
-                                            AppConstants.getPaths().get("APPLICATION_TEMPLATES")
-                                        )
-                                    }
-                                    onSearchQueryClear={ handleSearchQueryClear }
```

Comment:
The custom modal used in google and SIWE connections had the search bar. Need to bring it to the common `ApplicationSelectionModal`

Reply:
This will be tracked as a separate improvement with https://github.com/wso2/product-is/issues/20791